api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -2646,6 +2646,15 @@
             ]]>
           </description>
         </field>
+        <field>
+          <name>priority</name>
+          <version>4.2.0+</version>
```

Comment:
thinking at it, if we keep it as a XML element, it should be version 5.0.0, not 4.2.0
and associated to Maven 5
= https://cwiki.apache.org/confluence/display/MAVEN/Maven+Core+Roadmap

Reply:
Could you clarify on the ML what you consider breaking and non breaking ?  I don't think we're on the same page, so this clearly needs some clarification.